app/models/container.rb

```diff
@@ -24,6 +24,8 @@ class Container < ApplicationRecord
   include EventMixin
   include Metric::CiMixin
 
+  delegate :created_on, :to => :container_group
+
```

Comment:
This solved a CB bug. I can move this to a different PR.

Reply:
cc @simon3z

Reply:
@zeari it would be preferable to have this in another PR, easier to track and merge (hopefully?).

Reply:
sure